script/openqa-bootstrap-container

```diff
@@ -1,11 +1,18 @@
-#!/bin/bash -x
+#!/bin/bash
+set -e
```

Comment:
Why not just `#!/bin/bash -ex` here? I see that you add `set -x` later, both works

Reply:
This was done so that the error message about not having root rights would be shown more cleanly.